### DISCUSSION TITLE
Can't change SD Chip Select Pin in latest CI build

### DISCUSSION THREAD

**Discussion by jasonsomers** [forkineye/ESPixelStick#684] [Q&A]
Hello all.  Flashed the D1 Mini 32 along with the D1 Mini's latest CI build 4.0-ci6750955713 and I have noticed that I cannot change the value of the SD Chip Select Pin. When I change it and  save, then navigate away and then back, it still shows the original pin.
I tried several of the other non-Ethernet builds with the same results.
Anyone else have this issue?
Thanks,
Jason

**Comment by MartinMueller2003** [comment]
please post a screen capture of the admin page

**Reply by jasonsomers** [reply]
![admin_page](https://github.com/forkineye/ESPixelStick/assets/121630262/c9a22d8f-7c74-4c75-a020-2f0066f69f51)

**Reply by jasonsomers** [reply]
[https://youtu.be/JTH_4rUKf00](url)

**Comment by jasonsomers** [comment]
Just a quick follow up - it also appears that none of the settings in that section can be altered either.
(MOSI, MISO, CLK or Blank Time, as well as CS)

**Comment by MartinMueller2003** [comment]
Fixed in latest PR 685
https://github.com/MartinMueller2003/ESPixelStick/releases/tag/dist
